server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -21,21 +21,29 @@ import scala.util.control.NoStackTrace
   * @param filter To filter which assets from the webjars should be served
   * @param cacheStrategy strategy to use for caching purposes.
   * @param classLoader optional classloader for extracting the resources
+  * @param preferGzipped prefer gzip compression format?
   */
 class WebjarServiceBuilder[F[_]] private (
     blocker: Blocker,
     webjarAssetFilter: WebjarServiceBuilder.WebjarAssetFilter,
     cacheStrategy: CacheStrategy[F],
-    classLoader: Option[ClassLoader]) {
+    classLoader: Option[ClassLoader],
+    preferGzipped: Boolean) {
```

Comment:
If we added a secondary constructor here, this could be binary compatible and added to 0.21. What do you think?

Reply:
series/21 branch does not have `WebjarServiceBuilder`. It just has a `WebjarService` object - https://github.com/http4s/http4s/blob/series/0.21/server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

Reply:
Oops, bad memory/assumption on my part.